restapi/api/util.go

```diff
@@ -10,16 +10,16 @@ import (
 	"time"
 
 	"github.com/gin-gonic/gin"
-	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
 )
 
 type GenericResponse struct {
 	Message string `json:"message,omitempty"`
 	Error   string `json:"error,omitempty"`
 }
 
-//GetVersion reads the contents of the file version.txt and returns it.
-//If the file cannot be read, it returns "could not read version"
+// GetVersion reads the contents of the file version.txt and returns it.
```

Comment:
While I don't have a problem with adding these whitespaces in general, it usually is a good practice not to mix code cleanup/automated refactoring with your changes. It makes it hard to review and is a risk of adding merge conflicts. Please remove them from this PR, I am ok with merging them separately.